allensdk/brain_observatory/behavior/behavior_project_cache/tables/util/metadata_parsers.py

```diff
@@ -0,0 +1,144 @@
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+
+#####################################################################
+#
+#                   VBO PROJECT CONSTANTS
+#
+#####################################################################
+
+NUM_STRUCTURES_DICT = {
```

Comment:
1 More thing. This ophys metadata doesn't really belong in this file, which is specific to parsing. Also it is ophys specific. Can it be moved to something like ophys/project_constants ?